### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by mattretzer**
I ran this through the test_MS on staging..  this PR should trigger a test too :)

### CODE REVIEW

**Comment by mattretzer** [comment]
I guess this didn't trigger a travis test, since it's a PR into a parent branch that doesn't have Travis enabled?  Requesting review!

File: core/utilities/mcmlln-tools.rb [hunk 1]
```diff
@@ -63,16 +63,23 @@ def self.dirListFiles(directory)
...
+        json_hash={}
+      end
       json_hash
     end
 
     def self.write_json(hash, json)
-      finaljson = JSON.pretty_generate(hash)
-      File.open(json, 'w+:UTF-8') { |f| f.puts finaljson }
+    	#the 'unless' prevents Travis from erroring on writing json_log file at end of every script
+  		unless ARGV.empty?
```
Comment (nelliemckesson) [inline core/utilities/mcmlln-tools.rb]: Why is this spacing weird?

File: unit_tests.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    no_endnotes = '<section data-type="appendix" class="abouttheauthor" id="d1e4739"><h1 class="BMHeadbmh">About the Author</h1><p class="BMTextNo-Indentbmtx1"><span class="spansmallcapscharacterssc">Kai Ashante Wilson</span>&#x2019;s stories &#x201C;Super Bass&#x201D; and &#x201C;The Devil in America,&#x201D; the latter of which was nominated for the Nebula, the Shirley Jackson, and the World Fantasy Awards, can be read online gratis at <em class="spanitaliccharactersital">Tor.com</em>. His story &#xAB;<em class="spanitaliccharactersital">L&#xE9;gendaire</em>.&#xBB; can be read in the anthology <em class="spanitaliccharactersital">Stories for Chip,</em> which celebrates the legacy of science fiction grandmaster Samuel R. Delany. Kai Ashante Wilson lives in New York City.</p></section></body></html>'
+
+    assert_equal(endnotes_with_content, stripEndnotes(endnotes_with_content))
```
Comment (nelliemckesson) [inline unit_tests.rb]: Can you add some comments to say what each of these tests are doing?

**Comment by mattretzer** [comment]
OK @nelliemckesson, I made the updates!

**Pull request merged**